pull request Maioli Jacopo

JacopoMaioli opened the pull request.

JacopoMaioli committed:
prova

prova

JacopoMaioli committed:
add Lifestyle: Smoking in tabacco use , in sheet patient,

JacopoMaioli committed:
add lifestile smoking in smoking , in  sheet patient

JacopoMaioli committed:
Revert "add Lifestyle: Smoking in tabacco use , in sheet patient,"

This reverts commit 4c7d56505c6efd040d6b4f9bb12637a9dfd361b7.

JacopoMaioli committed:
update cell M11 sheet Diagnosis

update cell M11 sheet Diagnosis

JacopoMaioli committed:
update name value sett cell m11 sheet Diagnosis

cancer diagnosis to cancer hystology subgroup

JacopoMaioli committed:
Merge branch 'master' of https://github.com/JacopoMaioli/idea4rc

JacopoMaioli committed:
correction cell M11

ho lasciato degli spazi nell'update della cella M11 , qui ci sono le correzioni

JacopoMaioli committed:
creation cancer hystology , from sheet Diagnosis cell M9 sheet Diagnosisl

JacopoMaioli committed:
creation sites from cell m10 sheet diagnosis

JacopoMaioli committed:
update cell 12 , sheet diagnosis

JacopoMaioli committed:
correzione errori

JacopoMaioli committed:
create Cancer Diagnosis

create cancer diagnosis from sheet HNC condition list . sarcome condition list

JacopoMaioli committed:
update description cancer diagnosis

JacopoMaioli committed:
raggruppamento con commento di cancer hystology sub group

JacopoMaioli committed:
creation CodedtestResult.fsh

cretion Vs vodedtestedresult.fsh
formed by  MitoticIndexVS
add mitotic index in vs lab test
and some correction in cancer hystology

JacopoMaioli committed:
update subsites.fsh

update subsite.fsh , i rewrite all codes from cellM12 sheet Diagnosis and comment the other codes

JacopoMaioli committed:
unpdate cancerhystologysubgroup

JacopoMaioli committed:
creation extension for primary cancer

JacopoMaioli committed:
update 36768749 Invasion into the fascia  , in VS Simple measures

rgazzarata committed:
Merge branch 'idea4rc2025' into master

rgazzarata commented on input/fsh/extensions/extensions.fsh:
Metti come descrizione "This extension provides a mean to indicate the histology behaviours of a condition."

rgazzarata replied:
Metterei come cardinalità 1..1
Cambierei leggermente la descrizione: ora sono uguali ma metterei al posto di behaviours in un caso grup e nell'altro subgroup

rgazzarata replied:
Qui se non sbaglio abbiamo detto di di usare una sola extension e fare il bindign ad un vs che contenga entrambi i vs (come per site)

rgazzarata replied:
Vs: CancerHistologyBehaviorVS (padre) che contiene CancerHistologyVS e CancerHistologySubgroupVS

rgazzarata commented on input/fsh/profiles/ConditionCancer.fsh:
L'estension devi inserirla sotto CancerConditionCommonRules

rgazzarata replied:
Dovresti metterlo sotto RuleSet: CancerConditionCommonRules alla riga 18

rgazzarata replied:
Metterei i 2 elementi histology-morphology-behavior e histology-morphology-subgroup-behavior con cardinalità 0..1
Per side e subsite, vediamo cosa dice Giorgio, per ora commenta, non credo che l'estensione vada bene in questo caso.

rgazzarata replied:
Da completare come indicato

rgazzarata replied:
Anche qui l'elemento deve essere uno solo per histology con cardinalità 2 -< foerse non va qui ma direttamente nel profilo
Idem per site (va anche aggiornato il bindiang al site, anche se mi pare che sia già ok)

rgazzarata replied:
Anche il bodysite dovrebbe avere cardinalità 0..2

JacopoMaioli committed:
correction extension  histology Behavior

correction extension  histology Behavior
and correction hystology in histology

JacopoMaioli committed:
Merge branch 'master' of https://github.com/JacopoMaioli/idea4rc

JacopoMaioli committed:
update Model Overview

JacopoMaioli committed:
change Histology-behavior in Histology morphology behavior

change of name , and fix the location

JacopoMaioli committed:
add extension HistologyMorphologysubgroupBehavior

JacopoMaioli committed:
update observation: test result

JacopoMaioli committed:
add site and subsite like extension in bodySite

rgazzarata commented on input/images-source/AdverseEvent-map.plantuml:
Perchè hai eliminato @startuml? Sono abbastanza certa vada lasciato

rgazzarata commented on input/fsh/profiles/Observations/ObservationSimpleResult.fsh:
obs-valueOftype-quantity -> Devi cambiare la descrizione aggiungendo il nome del nuovo esame.
obs-valueOftype-codeable -> Devi cambiare la descrizione aggiungendo il nome del nuovo esame.
obs-valueOftype-codeable -> è un ID quindi deve essere univoco, devi quindi cambiare il nome aggiungendo in uno per esempio micotic e nell'altro positive-negative
obs-valueOftype-codeable -> Devi cabiare in entrambi la descrizione che rappresenta la descrizione a parole di quanto è scritto in pseudo-codice

rgazzarata replied:
Da aggionare

rgazzarata replied:
I contrains devono essere univoci, quindi devi aggiornare il nome in modo che siano univoci
Gli ultimi 2 contrains vanno rivisti nella descrizione mettendo che se.... allora il value deve essere tra i valori del vs (mettendo il nome del vs)

JacopoMaioli committed:
update file  excel models and maps

JacopoMaioli committed:
correction file AdverseEvent-map.plantuml

add startmul in the file

JacopoMaioli committed:
correction cardinality histology-morphology-behavior  e histology-morphology-subgroup-behavior

JacopoMaioli committed:
change extension site and subsite in comment

JacopoMaioli committed:
correction file input/fsh/profiles/Observations/ObservationSimpleResult.fsh

rgazzarata commented on input/fsh/model-maps/Diagnosis2FHIR.fsh:
Da rivedere appena chiudiamo i profili

rgazzarata commented on input/fsh/model-maps/DiagnosisI4RC.fsh:
Da aggiornare

rgazzarata replied:
Anche l' histologySubgroup deve essere opzionale (histology e site e i corrispettivi sub devono essere optionali perchè non si sa quali delle 2 copie possono esserci e quali no.)
La diagnosisCode deve essere invece obbligatoria

rgazzarata replied:
Forse devo ancora finire, ma anche histologySubgroup dovrebbe avere cardinalià 0..* (potresti avere il group o/e il subgroup).
Non vedo se la diagnosi è obbligatoria

rgazzarata commented on input/fsh/model-maps/Subject2FHIR.fsh:
Verificare che sia corretto

rgazzarata commented on input/fsh/model-maps/SurgeryI4RC.fsh:
Da controllare

rgazzarata replied:
I commenti di Giorgio lasciali
BackboneElement lasciali

rgazzarata replied:
Lascia i commneti di Giorgio.

JacopoMaioli committed:
update constrains

JacopoMaioli committed:
chek profile , clinical stage , disease extend , pathological stage , subject , surgery ,

i profili per quanto riguarda il varieble name , database , objetcclass , ObjectPropertyLabelEN,DataElementConceptDefEN,FormatConceptualDomain ,Required , comaciano con il foglio excel , a eccezione di quelli modificati , ho notato che l'unica modifica sono è per i VN con FormatConceptualDomain backboneElement , che su il foglio excel sono riportati in boolean , o non sono presenti nel foglio excel , nello specifico , il VN sistes in ClinicalStageI4RC.fsh,il VN sistes in DiseaseExtentI4RC.fsh,  siteOfSurgeryOnMetastasis in SurgeryI4RC.fsh

rgazzarata commented on input/fsh/model-maps/ClinicalStageI4RC.fsh:
No va lasciato booolean, ti spiego a voce.

rgazzarata commented on input/fsh/model-maps/DiseaseExtentI4RC.fsh:
Sites va lasciato, stesso discorso per tutti i BackboneElement

rgazzarata commented on input/fsh/model-maps/PathologicalStageI4RC.fsh:
Anche qui BackboneElement vanno lasciati

rgazzarata commented on input/fsh/model-maps/SubjectI4rc.fsh:
Idem BackboneElement

JacopoMaioli committed:
creation vs cancerhistolgybehaviour that contai , cancerHistology and cancerhystologysubgroup

JacopoMaioli committed:
Revert "chek profile , clinical stage , disease extend , pathological stage , subject , surgery ,"

This reverts commit e10a5833fa096c758057d533758b8d159b8e8008.

JacopoMaioli committed:
update profile diagnosis

histology group , sub gruop in Optional
site sub site in optional
code in mandatory

JacopoMaioli committed:
upadate , observation , discription , and obs

JacopoMaioli committed:
update cancerhistologybehavior

JacopoMaioli committed:
update , exetension site and Histologymorphologybehavior

JacopoMaioli committed:
upgrade SurgeryI4RC.fsh,DiagnosisI ConditionCancer.fsh4RC.fsh
